lib/bitmap/bitmap.c

```diff
@@ -152,6 +152,7 @@ int BM_set_mode(int mode, int size)
     case BM_FLAT:
     case BM_SPARSE:
         Mode = mode;
+        break;
```

Comment:
This changes the logic. However, the function `BM_set_mode()` is not used anywhere. No code to break (pun intended:).